x-pack/plugins/alerting/server/application/maintenance_window/methods/find/types/find_maintenance_window_result.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 import { TypeOf } from '@kbn/config-schema';
-import { findMaintenanceWindowsResultSchema } from '../schemas';
+import { findMaintenanceWindowsResultSchema, findMaintenanceWindowsParamsSchema } from '../schemas';
 
 export type FindMaintenanceWindowsResult = TypeOf<typeof findMaintenanceWindowsResultSchema>;
+export type FindMaintenanceWindowsParamsSchema = TypeOf<typeof findMaintenanceWindowsParamsSchema>;
```

Comment:
We already do that in `x-pack/plugins/alerting/server/application/maintenance_window/methods/find/types/find_maintenance_window_params.ts`, right?